example/config-overrides.js

```diff
@@ -0,0 +1,10 @@
+const { override, addWebpackAlias } = require('customize-cra')
+
+const path = require('path')
+
+// This is needed in order to correctly resolve React versions between the library & example app
+module.exports = override(
```

Comment:
Is this going to be needed for any other (eventual) shared dependencies?

Reply:
It _shouldn't_, as far as I'm aware `react` notably causes issues if multiple instances are loaded and that's what this workaround is for.